osu.Game.Rulesets.Osu/Mods/OsuModApproachDifferent.cs

```diff
@@ -86,6 +89,7 @@ private Easing getEasing(AnimationStyle style)
 
         public enum AnimationStyle
         {
+            Linear,
```

Comment:
I feel like this should be "None" because I don't think "Linear" is friendly to user

Reply:
you may be right, ill wait for a second opinion and change it if your suggestion is favoured

Reply:
Disagree. The related setting is described as the "animation style of the approach circles". Therefore it makes no sense to put "none" there as an option because that just reads as if there was no animations at all, and what would that mean? No approach circles, or just static ones? So I would say "none" is actively misleading in this context.

Reply:
im thinking "classic" or "standard"  might be good options, thoughts?

Reply:
I'm fine with keeping linear. Standard might also be ok, not sure.